primitives/src/array.rs

```diff
@@ -33,12 +33,32 @@ construct_hash_bytes! {
     pub struct H128(16);
 }
 
+construct_hash_bytes! {
+    #[derive(Clone)]
+    pub struct H160(20);
+}
+
+construct_hash_bytes! {
+    #[derive(Clone)]
+    pub struct H256(32);
+}
+
 construct_hash_bytes! {
     #[derive(Clone)]
     pub struct H512(64);
 }
 
 construct_hash_bytes! {
     #[derive(Clone)]
-    pub struct H1024(256);
```

Comment:
Is this a bug?

Reply:
H1024 has 128 bytes, the name is not right.